src/messages/forms.messages.js

```diff
@@ -22,32 +22,16 @@ const formsMessages = defineMessages({
     defaultMessage: 'Select permission'
   },
   orderProcessName: {
-    id: 'formMessages.orderProcessName',
+    id: 'forms.orders.processName',
```

Comment:
The order and order process are two different things. Order-processes should be the namespace (or something similar).

Reply:
moved!